.github/workflows/generate-changelog.yml

```diff
@@ -13,11 +13,48 @@ jobs:
     with:
       script_call: 'make tools update-changelog-unreleased-section'
       file_to_commit: 'CHANGELOG.md'
-      commit_message: ${{ github.event_name == 'workflow_dispatch' && 'Update CHANGELOG.md (Manual Trigger)' || format('{0}{1}', 'Update CHANGELOG.md for \#', github.event.pull_request.number) }}
+      commit_message: ${{ github.event_name == 'workflow_dispatch' && format('{0}{1}', 'chore:',' Updates CHANGELOG.md (Manual Trigger)') || format('{0}{1}{2}{3}', 'chore:', ' Updates CHANGELOG.md for ', '#', github.event.pull_request.number) }}
```

Comment:
":" and "#" are problematic in yaml files ([reference](https://github.com/actions/runner/issues/1019#issuecomment-808880611)), format method  makes it work

Reply:
can't 'chore:' go in the same param as CHANGELO... ?